auth-web/src/util/common-util.ts

```diff
@@ -159,24 +159,25 @@ export default class CommonUtils {
 
   static getAdminPermissions (): string[] {
     return [
```

Comment:
added *Permission.CHANGE_ADDRESS* and *Permission.CHANGE_ORG_NAME* and then sorted into alphabetical order to make it easier to see what permissions are there/not there